ec/acc/testdata/extension_bundle_download.tf

```
@@ -0,0 +1,6 @@
+resource "ec_extension" "my_extension" {
+  name           = "%s"
+  version        = "*"
```

Should be the exact version, in this case `7.10.1`:
```suggestion
  version        = "7.10.1"
```

---

fixed by d340fa3